src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/IepSummary.kt

```diff
@@ -24,17 +25,19 @@ data class IepSummary(
   val iepDate: LocalDate,
   @Schema(description = "Date and time when last review took place", required = true, example = "2021-12-31T12:34:56.789012")
   val iepTime: LocalDateTime,
-  @Schema(description = "Days Since last Review", example = "23", required = true)
-  val daysSinceReview: Int,
   @Schema(description = "Location  of prisoner when review took place within prison (i.e. their cell)", example = "1-2-003", required = false)
   val locationId: String? = null,
   @Schema(description = "IEP Review History (descending in time)", required = true)
   val iepDetails: List<IepDetail>,
 ) {
 
-  fun daysSinceReview(): Int {
-    return daysSinceReview(iepDetails)
-  }
+  @get:Schema(description = "Days since last review", example = "23", required = true)
+  @get:JsonProperty
+  val daysSinceReview: Int
+    get() {
```

Comment:
How is this `get() { }` works? Is it a getter for the `val` immediately above?

And presumably this is why you can now replace `.daysSinceReview()` calls with just `.daysSinceReview`?